### PR TITLE
BUGFIX: Make close button in Image preview work

### PR DESCRIPTION
**Pull request by rolandschuetz**
When a user navigates to the single view of an asset by clicking on the thumbnail in the inspector,
the close button of the single will work again.

NEOS-1871 #close

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -9,6 +9,10 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
+[*.js]
```
Comment (kitsunet) [inline .editorconfig]: Why this, it has nothing to do with the rest of the PR.

Also should be discussed if we want to keep tabs here, makes switching contexts between PHP and JS harder IMHO.

**Comment by kitsunet** [comment]
Actual change makes sense IMHO, but someone else should have a look.

File: .editorconfig (hunk 1 above)
Comment (rolandschuetz) [inline .editorconfig]: True, that could have been a separate PR.

All JavaScript files are right now using exactly this configuration. When you look into the code, they use tabs. My editor was picking up on the wrong config and was doing bad stuff...

**Comment by kdambekalns** [comment]
Is this fixing https://jira.neos.io/browse/NEOS-1871?

Hard to tell, since the is no description… please, write meaningful commit/PR messages!

**Comment by rolandschuetz** [comment]
"BUGFIX: If the media editor is reopened, reset to initial state"

I didn't know the ticket, this pull request would solve exactly that issue. https://jira.neos.io/browse/NEOS-1871

I discovered this behavior is not so important anymore, since there were changes in 2.3.

File: TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js [hunk 2]
```diff
@@ -1025,10 +1025,35 @@ function (Ember, $, FileUpload, template, cropTemplate, BooleanEditor, Spinner,
...
+			var that = this;
+			SecondaryInspectorController.addObserver('_visible', function(controller) {
+				if(that.get('_mediaBrowserView') !== controller.get('_viewClass') || controller.get('_visible')) {
+					return;
+				}
+				var iframe = $('#'+mediaViewDomId);
+				// src attribute is not updated, get the current href from the iframe
+				if(iframe[0].contentWindow.location.href !== url) {
+					iframe.attr('src', url);
+				}
```
Comment (aertmann) [inline TYPO3.Neos/Resources/Public/JavaScript/Content/Inspector/Editors/ImageEditor.js]: we use whitespace a little different.. e.g. `if (` and `'#' + media`

**Comment by aertmann** [comment]
actually this issue is fixed in #532

**Comment by rolandschuetz** [comment]
Feel free to close the issue.

**Comment by robertlemke** [comment]
Okay, should we close this issue then and make sure to test & merge #532 soon?

**Comment by aertmann** [comment]
just be aware that #532 only targets 2.3, if a fix in 2.1 is wanted feel free to push re-open or push a new one

**Pull request closed without merging**